test/downstream/observables_autodiff.jl

```diff
@@ -46,6 +46,15 @@ sol = solve(prob, Tsit5())
     @test du == gs
 end
 
+# @testset "AD Observable Functions for Initialization" begin
+#     iprob = prob.f.initialization_data.initalizeprob
+#     isol = solve(iprob)
+#     gs, = gradient(isol) do isol
+#         isol[w]
+#     end
+
+# end
```

Comment:
I assume this will be enabled later?

Reply:
Yes,  this requires https://github.com/SciML/SciMLSensitivity.jl/pull/1168